app-text/booksorg/booksorg-0.3.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="A Simple Books Organizer"
+DESCRIPTION="Application to Organize with ease your pdf book"
+COPYRIGHT="2014-2018 Abou Zakaria"
+LICENSE="GNU GPL v3"
+REVISION="1"
+HOMEPAGE="http://booksorg.sourceforge.net/"
+SOURCE_URI="https://downloads.sourceforge.net/booksorg/booksorg_${portVersion/./}.tar.gz"
+SOURCE_DIR="booksorg_${portVersion/./}"
+CHECKSUM_SHA256="dcdcce89090d1539c020412d294e2f6c0ffc84a0f4f400132f57e1c1be29d234"
+PATCHES="booksorg-$portVersion.patchset"
+ADDITIONAL_FILES="booksorg.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="?x86"
```

Comment:
```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 if [ "$targetArchitecture" = x86_gcc2 ]; then
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
 fi
```
(You enabled x86_64 and x86 primary arch but forgot to re-enable x86 secondary arch. I can do it for you if you wish.)

Reply:
ah.. i'm forget that.. please do i you not mind.. thanks

Reply:
No problem. Don't worry ;)